macros/src/lib.rs

```diff
@@ -61,9 +56,24 @@ fn try_test(attr: TokenStream, input: ItemFn) -> syn::Result<Tokens> {
   let logging_init = expand_logging_init(&attribute_args);
   let tracing_init = expand_tracing_init(&attribute_args);
 
+  let (inner_test, generated_test) = if attr.is_empty() {
+    let test_attr: syn::Attribute = parse_quote! { #[::core::prelude::v1::test] };
```

Comment:
Type annotation does not seem to be necessary, right?
```suggestion
    let test_attr = parse_quote! { #[::core::prelude::v1::test] };
```